web/help/rlt-notes-unit1.md

```diff
@@ -0,0 +1,14 @@
+If you do not leave a space before and after the & the popup does not show
+If the student makes change and clicks an example they get a warning. It might be beneficial to mention something about this and/or saving.
+
+Should variable really be constant since you cannot re-assign it a new value?
+Ex. program=xxx
+nameTag=xxx
+nameTag=yyy
+
+Might want to mention the red x error shows when using a variable before it's defined.
```

Comment:
Sure, if there's somewhere that it would help to explain how the system indicates errors, go for it.  I think you should focus first on explaining the bigger idea, that errors will often be underlined in red as you type, and then explain the specifics of that idea from that basis.

Reply:
I'll give it some thought.